### ISSUE TITLE
Visible & Hidden do not seem to follow custom breakpoints. 

### ISSUE THREAD

**Issue by monishkumar7** [santosfrancisco/react-awesome-styled-grid#125]
I have set custom breakpoints - only 4 (sm - 0-599, md - 600-899, lg - 900-1199 and xl - 1200+) along with a html font size of 62.5. So, in the customConf, the values for breakpoint are 1, 37.5, 56.25 and 75. Visible & Hidden seem to work with 4 breakpoints but they start from xs and not sm as specified. Also, it ignores the html font-size I set using GlobalStyles. So, Visible & Hidden work at 375px, 562px & 750px.

**Comment by monishkumar7** [comment]
I have set custom breakpoints - only 4 (sm - 0-599, md - 600-899, lg - 900-1199 and xl - 1200+) along with a html font size of 62.5. So, in the customConf, the values for breakpoint are 1, 37.5, 56.25 and 75. Visible & Hidden seem to work with 4 breakpoints but they start from xs and not sm as specified. Also, it ignores the html font-size I set using GlobalStyles. So, Visible & Hidden work at 375px, 562px & 750px.
